cli/mne_bids_raw_to_bids.py

```diff
@@ -55,13 +56,28 @@ def run():
                       action='store_true')
 
     opt, args = parser.parse_args()
+    opt_dict = vars(opt)
 
+    # Get the format of the raw data
+    _, ext = _parse_ext(opt_dict['raw_fname'])
+
+    # Prepare a read_params dict
+    if ext in ['.con', '.sqd']:
+        read_params = dict(elp=opt.electrode, hsp=opt.hsp, hpi=opt.hpi)
+    elif ext == '.pdf':
+        read_params = dict(config_fname=opt.config, head_shape_fname=opt.hsp)
```

Comment:
you shouldn't have to specify them. It should come from BIDS

Reply:
this gets passed to _read_raw so it's not yet in BIDS AFAIK.

Reply:
This is the `write` utility of the CLI ... so the data is not yet in BIDS and has to be supplied "manually"